script.sql

```diff
@@ -104,15 +104,15 @@ select
 	order_amount,
     ex_rate,
     currency,
-    round((order_amount * ex_rate),1)  as 'lifetime_purchase'  
```

Comment:
@Iriakuma this looks good, however please remove the duplicate whitespace o 107 and 108.

Reply:
Done